### PR TITLE
Check destination address of DCP get/set requests

### PR DESCRIPTION
**Pull request by olbjo**
We should rerun tests before merging!

### CODE REVIEW

File: src/common/pf_dcp.c [hunk 1]
```diff
@@ -950,15 +974,15 @@ static int pf_dcp_get_set (
       if (
          (p_rsp != NULL) &&
          (pf_dcp_check_sam (net, &p_src_ethhdr->src) == true) &&
-         ((p_src_ethhdr->dest.addr[0] & 1) == 0)) /* Not multi-cast */
+         (pf_dcp_check_destination_address (net, &p_src_ethhdr->dest) == true))
```
Comment (pyhys) [inline src/common/pf_dcp.c:977]: If you like, you can remove `== true`

File: src/common/pf_dcp.c [hunk 2]
```diff
@@ -271,6 +271,30 @@ static bool pf_dcp_check_sam (pnet_t * net, const pnet_ethaddr_t * mac)
           (memcmp (&net->dcp_sam, mac, sizeof (net->dcp_sam)) == 0);
 }
 
+/**
```
[the remark this answers is not in the document]
Comment (olbjo) [reply src/common/pf_dcp.c:274]: OK, 
used suggestion but changed "destination" to "destination_mac". 
Added tests.
Done

File: src/common/pf_dcp.c (hunk 1 above)
Comment (olbjo) [reply src/common/pf_dcp.c]: Done

**Review by pyhys — APPROVED** [review]
Great!

**Review by pyhys — APPROVED** [review]
Great!

**Pull request merged**